lib/server.js

```diff
@@ -1359,7 +1359,7 @@ class WebServer {
                             reject(err);
                         } else {
                             if (attr_Sat) {
-                                this.adapter.setForeignState(attr_Sat.setId, s, false, {user}, err => err ? reject(err) : resolve());
+                                this.adapter.setForeignState(attr_Sat.setId, s / 100 * attr_Sat.max, false, {user}, err => err ? reject(err) : resolve());
```

Comment:
Does `attr_Sat.min` exists too?

Reply:
No, currently not. It is set here: https://github.com/Garfonso/ioBroker.lovelace/blob/hue/lib/server.js#L1592 
We could add attr_Sat.min, too. But usually it is 0 = white/black (i.e. no color at all). Other values as minimum do not make much sense. Max is a different thing and is usually 100 if sat is a percentage or 255 as byte value.